runfiles_test.go

```diff
@@ -101,3 +101,31 @@ func TestRunfiles_empty(t *testing.T) {
 		t.Errorf("Path for empty file: got error %q, want something that wraps %q", got, want)
 	}
 }
+
+var manifestWithDirTestCases = map[string]string {
+	"foo/dir": "path/to/foo/dir",
+	"foo/dir/file": "path/to/foo/dir/file",
+	"foo/dir/deeply/nested/file": "path/to/foo/dir/deeply/nested/file",
+}
+
+func TestRunfiles_manifestWithDir(t *testing.T) {
+	dir := t.TempDir()
+	manifest := filepath.Join(dir, "manifest")
+	if err := os.WriteFile(manifest, []byte("foo/dir path/to/foo/dir\n"), 0600); err != nil {
+		t.Fatal(err)
+	}
+	r, err := runfiles.New(runfiles.ManifestFile(manifest), runfiles.ProgramName("/invalid"), runfiles.Directory("/invalid"))
```

Comment:
I think the `/invalid` stuff is no longer necessary now that the library prefers using `ManifestFile` if set